projects/frontend/shared-components/gui/.npmrc

```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```

Comment:
maybe an additional post-merge PR, to explain in the VEP/README what we have here

Reply:
This flag reverts to the previous npm behavior of not installing peer deps by default. Not sure why we have this.

Reply:
What should we add to the VEP that's related to the flag?

Reply:
https://github.com/vmware/versatile-data-kit/issues/1643